docs/streamtypes/caip-10-link/api.md

```diff
@@ -0,0 +1,126 @@
+# Caip10Link API
+
+This guide demonstrates how to create, update, and query Caip10Links on the Ceramic network using the [HTTP](../../../reference/javascript/clients) and [core](../../../reference/javascript/clients) clients.
+
+## Prerequisites
+You need an [installed client](../../build/installation.md) and an [authenticated user](../../build/authentication.md) to perform writes to Caip10Links on the network during runtime. If you only wish to query existing Caip10Links then you still need an installed client but it doesn't need to be authenticated.
```

Comment:
You don't?  Even to set the DID, there's no signature from the DID involved?  Does that mean that the owner of a blockchain address could create a Caip10Link linking the blockchain address to a DID they don't actually control?

Reply:
Yes that's how it works currently. If you want to verify both ways you should make sure that the caip10-link is listed in the *cryptoAccounts* IDX definition.

Reply:
In theory it would be possible to change Caip10Links to additionally sign the link proof with the linked DID when setting the DID link, right?  Then while only the blockchain->DID link would be *discoverable* without IDX, both the blockchain->DID AND DID->blockchain link would be *verifiable* from just the Caip10Link.  Users who wanted to prove bidirectional linkage could still manually pass around the StreamID without using IDX

Reply:
I filed https://github.com/ceramicnetwork/js-ceramic/issues/1354 to follow-up on this idea. In the meantime I updated the docs here to remove the requirement of an authenticated client to write Caip10Links.

Reply:
It's possible to implement. Not sure which UX considerations are needed for that.